src/main_app/admin/flask_admin_panel_models.py

```
@@ -0,0 +1,74 @@
+        models=[
+            JobRecord,
+            SettingRecord,
+        ],
+    ),
+]
+
+
+__all__ = [
+    "categories",
+    "ModelItem",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)